test/openAPI/features/applicant_applications_fileId.feature

```diff
@@ -0,0 +1,22 @@
+@method=POST @endpoint=/applications/{fileId}
```

Comment:
Please change the file name `applicant_applications_fieldid` or `applicant_applications_field_id`

Reply:
I agree, but fileid -> not fieldid